src/mngt/ocf_mngt_cache.c

```diff
@@ -938,16 +935,38 @@ static void _ocf_mngt_attach_load_properties_end(void *priv, int error,
 	context->metadata.shutdown_status = properties->shutdown_status;
 	context->metadata.dirty_flushed = properties->dirty_flushed;
```

Comment:
Looks like these doesn't make any sense in attach. Can you please verify this? We can talk about this face to face